middleware/get_head.go

```diff
@@ -24,8 +24,7 @@ func GetHead(next http.Handler) http.Handler {
 			tctx.RouteMethod = "HEAD"
 			tctx.RoutePath = routePath
 
-			h := rctx.Routes.Find(tctx, tctx.RouteMethod, tctx.RoutePath)
-			if h == nil {
+			if !rctx.Routes.Match(tctx, tctx.RouteMethod, tctx.RoutePath) {
```

Comment:
Does `Match()` need three parameters? Seems like `RouteMethod` and `RoutePath` are already stored in `tctx` and available to the method.

Is there any use case where the `tctx` values would be different from method arguments?

Reply:
I believe the API is just clearer this way, as the routing context is intended to be read from, but can be manipulated to change logic, but shouldnt be an input in most common uses.